docs/user-guide/steerlm.rst

```diff
@@ -76,7 +76,13 @@ Untar the .nemo file to obtain the tokenizer in NeMo format (only for the 70B mo
 
 The prefix for the tokenizer would be different when extracted. Ensure that the correct tokenizer file is used when running the preceding command.
 
-Step 2: Download and Preprocess data for Attribute Prediction Modelling
+To follow the HelpSteer2 and HelpSteer2-Preference line of works, you would need to use the Llama 3 70B and Llama 3.1 70B Instruct models respectively.
+
+You need to get access to them, download and then convert them in a similar manner.
+
+An additional requirement is that you need to have an HF_TOKEN with access to the relevant models and then either export HF_TOKEN=<your_hf_token> or HF_HOME=<your_hf_home> contain a file named token with your token.
```

Comment:
This line can be removed. We're adding some documentation on `HF_TOKEN`s that will apply to all models

Reply:
sure done